lib/mission_control/jobs/console/commands.rb

```diff
@@ -0,0 +1,38 @@
+require "irb/command"
+
+module MissionControl::Jobs
```

Comment:
Looks like we might need to split this into two files, one for the `ConnectTo` class (`lib/mission_control/jobs/console/connect_to.rb`) and another one for the `JobsHelp` class (`lib/mission_control/jobs/console/jobs_help.rb`), to follow the naming conventions that Zeitwerk expects. I think, doing that, we won't need the call to `require "mission_control/jobs/console/commands"`, as Zeitwerk will take care of that for us.

Reply:
Thanks for the pointer! I wonder why I couldn't see the zeitwerk errors locally 🤔